valqueries-h2/src/main/java/com/valqueries/automapper/H2SqlDialect.java

```diff
@@ -59,10 +60,43 @@ public String limit(int offset, Integer limit) {
 				"    FETCH NEXT "+limit+" ROWS ONLY";
 	}
 
+	@Override
+	public String update(TypeDescriber<?> typeDescriber, List<Element> elements, List<Property.PropertyValue> newPropertyValues) {
+		StringBuilder updateStatement = new StringBuilder();
+
+		updateStatement.append("UPDATE " + getTableName(Clazz.of(typeDescriber.clazz())) + " as main SET ");
+
+		String columnsToUpdate = newPropertyValues.stream()
+				.map(pv -> "main." + sqlNameFormatter.column(pv.getProperty().getToken()) + " = :" + pv.getProperty().getToken().snake_case())
+				.collect(Collectors.joining(", "));
+		updateStatement.append(columnsToUpdate);
+
+		if (!elements.isEmpty()) {
+			updateStatement.append(" WHERE " + elements.stream().map(Element::queryString).collect(Collectors.joining(" AND ")));
+		}
+
+		return updateStatement.toString();
+	}
+
+	@Override
+	public String changeColumn(String table, String columnName, String sqlType) {
+		return null;
+	}
+
+	@Override
+	public String addIndex(String tablename, KeySet key) {
+		return null;
```

Comment:
or indexes?

Reply:
It doesn't support sql generator yet, no